### DISCUSSION TITLE
Forward only photos/videos

### DISCUSSION THREAD

**Discussion by ThatDevCoder** [aahnik/tgcf#38] [General]
First of all, thank you for the script!
Can you please create a feature that can specifically send photos or videos from a particular group. Because there is a lot of unnecessary stuff also forwarded from the group which we are forwarding. 
Moreover, if you keep running this script after forwarding like 10-15k messages, the telegram account gets instant banned. So please take note of that so that other's telegram account won't get banned.
And can you also create a feature that will only forward messages without a caption. That would be great if you could do that!

Anyways, thanks a lot, man!

**Comment by aahnik** [comment]
@ThatDevCoder  I understand the problem. In the future, support will be added for filtering by media type like photos/videos/gifs/pdf/stickers, etc.

Please see #2 for how to stay updated
